### PR TITLE
Update README.md for expo-cli api changes.

### LINKED ISSUE #35: Unable to resolve SVG file
Hi, I've been trying to import SVG file to use in my react-navigation file as an icon. I installed the library as you write in the readme file. But, I got an error.

`Unable to resolve "./assets/icons/bottom_tab/profile_active.svg" from "MainNavigation.js"
Failed building JavaScript bundle.`

I've been checked my file, and I'm really sure that the svg file is in right directory.

My environment: 
- expo: 33
- react-native-svg: 9.4.0
- react-native-svg-transformer: 0.13.0

I got this error on both android and ios.

### PR DESCRIPTION
**Pull request by chin8628**
Hi, I updated installation step for expo in README.md since the expo-cli api was changed follow to the solution from this issue https://github.com/expo/expo-cli/issues/875

close #35

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,8 +72,9 @@ If you are using [Expo](https://expo.io/), you also need to add this to `app.jso
 ```json
 {
   "expo": {
-    "packagerOpts": {
-      "config": "metro.config.js"
+    "packagerOpts": { 
+      "config": "metro.config.js", 
+      "sourceExts": ["js", "jsx", "svg", "svgx"]
```
Comment (kristerkari) [inline README.md]: The `.svgx` extension is only a fallback extension for older React Native versions, so it should not be included here as people should be using the standard `.svg` extension for the images.

**Comment by kristerkari** [comment]
Thanks a lot! I only had one small thing that could be changed.

File: README.md (hunk 1 above)
Comment (flexbox) [reply README.md]: Thank you for this PR @chin8628 
I added this 
```
"sourceExts": ["js", "jsx", "svg"]
```
and it's working for expo `"sdkVersion": "33.0.0",`

Comment (chin8628) [reply README.md]: @kristerkari I've removed 'svgx' extension in a new commit.

**Pull request merged**